game/tests.py

```
@@ -201,11 +202,176 @@ def test_duplicate_email_registration_fails(self):
+    """Test the cleanup_inactive_users management command."""
+
+    def test_cleanup_deletes_old_inactive_users(self):
+        from django.core.management import call_command
+        from django.utils import timezone
+        from datetime import timedelta
+
+        # Active user (should not be deleted)
+        User.objects.create_user(
+            username='active_user',
+            email='active@example.com',
+            password='StrongPass123!',
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "password"

(S106)